src/plugins/DevTools/components/InfoComponent.ts

```diff
@@ -27,6 +27,15 @@ export class InfoComponent extends OwlUIComponent<{}> {
     get reprStart(): string {
         return this._repr(this.state.range.start);
     }
+    /**
+     * Log a `VNode` to the console.
+     *
+     * @param {VNode} vNode
+     */
+    logVNode(vNode: VNode): void {
+        // Get from vDocument so we don't log a proxy
+        console.log(this.env.editor.vDocument.find(vNode.id));
```

Comment:
I realize I never actually *wrote* a comment summarizing the discussion we had about proxification, thus resulting in this PR looking like it is abandonned.

In short, since what you truly want to do is "unproxify" Owl's proxy, I would remove everything this PR adds to `VDocument` then put the code of `_findByID` in OwlUI and call it something like `unproxifyVNode` or `deproxifyVNode` depending on which one is more correct in english.

Or better still ! Do we truly need owl to *observe* the VNodes ? I mean, we know when they change since it triggers a refresh, and I don't think we ever do `node.something = ...` in the DevTools, so owl would not see the changes triggered by the editor itself anyway, so do we really need to put the VDocument in the observed state ?